SonarQube.Client.Tests/ChecksumCalculatorTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SonarQube.Client.Tests
+{
+    [TestClass]
+    public class ChecksumCalculatorTests
+    {
+        [TestMethod]
+        public void Checksum_ThrowsOnNull()
+        {
+            Action act = () => ChecksumCalculator.Calculate(null);
+
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("text");
```

Comment:
This test has changed slightly so it doesn't use the SLVS `Exceptions.Expect` helper class.